snuba/cli/api.py

```diff
@@ -7,13 +7,32 @@
 
 
 @click.command()
+@click.option("--bind", help="Address to listen on.")
 @click.option("--debug", is_flag=True)
 @click.option("--log-level", help="Logging level to use.")
 @click.option("--processes", default=1)
 @click.option("--threads", default=1)
-def api(*, debug: bool, log_level: Optional[str], processes: int, threads: int) -> None:
+def api(
+    *,
+    bind: Optional[str],
+    debug: bool,
+    log_level: Optional[str],
+    processes: int,
+    threads: int
+) -> None:
     from snuba import settings
 
+    port: Union[int, str]
+    if bind:
+        if ":" in bind:
+            host, port = bind.split(":", 1)
+            port = int(port)
+        else:
+            host = bind
+            port = settings.PORT
+    else:
+        host, port = "0.0.0.0", settings.PORT
```

Comment:
Could you please add a setting for the host? Since we allow customization of host and port, for consistency, it would make sense to have both manageable both via settings and command line.